crates/starknet-devnet-core/src/blocks/mod.rs

```diff
@@ -172,6 +174,27 @@ pub struct StarknetBlock {
     pub(crate) status: BlockStatus,
 }
 
+impl From<&StarknetBlock> for TypesPendingBlockHeader {
```

Comment:
We already have `impl From<&StarknetBlock> for TypesBlockHeader` - can this be handled in a better way in rust?

Reply:
no, the normal block has `block_hash` and `block_number` but pending don't

Reply:
and also `new_root`